monkey/monkey_island/cc/services/edge/displayed_edge.py

```diff
@@ -63,7 +63,7 @@ def get_pseudo_label(edge: Dict):
     @staticmethod
     def services_to_displayed_services(services, for_report=False):
         if for_report:
-            return [x for x in services]
+            return list(services)
```

Comment:
Why is this better if we're straight converting to a list?

Reply:
I do think this more clearly shows the intent of the code. If I want to understand what's being returned, I'll understand what `list()` is doing faster than if I have to parse the list comprehension.